ddtrace/commands/ddtrace_run.py

```diff
@@ -105,6 +104,15 @@ def main():
 
     log.debug("program executable: %s", executable)
 
+    if os.path.basename(executable) == "uwsgi":
+        log.error(
```

Comment:
I'd rather use here, if log is not configured you won't see anything. It's a command line tool after all.